File: 0682-baseball-game/0682-baseball-game.js
/**
 * @param {string[]} operations
 * @return {number}
 */
var calPoints = function(operations) {
  let answer = [];
  for (let oper of operations) {
    if (oper === '+') {
      let one = answer[answer.length - 1];
      let second = answer[answer.length - 2];
      answer.push(one + second);
    }
    else if (oper === 'C') {
      answer.pop();
    }
    else if (oper === 'D') {
      let one = answer[answer.length - 1];
      answer.push(one * 2);
    }
    else {
      answer.push(Number(oper));
    }
  }
  let sum = 0;
  for (let i = 0; i < answer.length; i++) {
  sum += answer[i];
  }
  return sum;
};

File: LeetCode/0682-baseball-game/0682-baseball-game.py
class Solution:
    def calPoints(self, operations: List[str]) -> int:
        answer_list = []
        for oper in operations:
            if oper == "C":
                answer_list.pop()
            elif oper == "D":
                answer_list.append(int(answer_list[-1]) * 2)
            elif oper == "+":
                answer_list.append(int(answer_list[-1]) + int(answer_list[-2]))
            else:
                answer_list.append(int(oper))

        return sum(answer_list)
        